src/rest_settings.ts

```diff
@@ -142,6 +142,108 @@ const paths = {
    * @method get
    */
   getCorrelationResults: `/correlations/${params.correlationId}/results`,
+  /*
+   * Waits for the ProcessStarted notification and returns the result
```

Comment:
Wie oben, auch für alle Routen.

```suggestion
   * Waits for a ProcessStarted notification and resolves with the notification's content-
```